tests/testimpl/types.go

```diff
@@ -0,0 +1,22 @@
+package testimpl
+
+import "github.com/launchbynttdata/lcaf-component-terratest/types"
+
+type ThisTFModuleConfig struct {
+	types.GenericTFModuleConfig
+	Vpc_id        string    `json:"vpc_id"`
+	Zone_name     string    `json:"zone_name"`
+	Comment       string    `json:"comment"`
+	Force_destroy bool      `json:"force_destroy"`
+	Records       []Records `json:"records"`
+}
```

Comment:
looks like some leftovers from an AWS module, 

this would probably be sufficient for this module:
```
package testimpl

import "github.com/launchbynttdata/lcaf-component-terratest/types"

type ThisTFModuleConfig struct {
	types.GenericTFModuleConfig
	// Empty: there are no settings for the skeleton module.
}
```